publish/dir.targets

```diff
@@ -1,33 +1,52 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <UsingTask TaskName="GenerateDebRepoUploadJsonFile" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll" />
+  <UsingTask TaskName="GenerateChecksums" AssemblyFile="$(LocalBuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.Local.dll" />
 
   <Target Name="GatherPublishToAzureBinaries">
-    <PropertyGroup>
-      <RelativeBlobPath>$(Channel)/Binaries/$(SharedFrameworkNugetVersion)</RelativeBlobPath>
-    </PropertyGroup>
     <ItemGroup>
       <PublishFile Include="@(CompressedFile)" />
       <PublishFile Include="@(RuntimePackageFile)" />
       <PublishFile Condition="'$(PublishRidAgnosticPackages)' == 'true'" Include="@(IdentityPackageFile)" />
       <Project Include="publish.proj">
         <PublishTarget>UploadToAzure</PublishTarget>
-        <AdditionalProperties>%(Project.AdditionalProperties);ItemsToPublish=@(PublishFile);$(OutputVersionBadge);RelativeBlobPath=$(RelativeBlobPath)</AdditionalProperties>
+        <AdditionalProperties>%(Project.AdditionalProperties);ItemsToPublish=@(PublishFile);$(OutputVersionBadge);RelativeBlobPath=$(BinariesRelativePath)</AdditionalProperties>
       </Project>
     </ItemGroup>
   </Target>
 
   <Target Name="GatherPublishToAzureInstallers">
-    <PropertyGroup>
-      <RelativeBlobPath>$(Channel)/Installers/$(SharedFrameworkNugetVersion)</RelativeBlobPath>
-    </PropertyGroup>
     <ItemGroup>
       <Project Include="publish.proj">
         <PublishTarget>UploadToAzure</PublishTarget>
-        <AdditionalProperties>%(Project.AdditionalProperties);ItemsToPublish=@(InstallerFile);RelativeBlobPath=$(RelativeBlobPath)</AdditionalProperties>
+        <AdditionalProperties>%(Project.AdditionalProperties);ItemsToPublish=@(InstallerFile);RelativeBlobPath=$(InstallersRelativePath)</AdditionalProperties>
+      </Project>
+    </ItemGroup>
+  </Target>
+  
+  <Target Name="GatherPublishToAzureChecksums" DependsOnTargets="CreateChecksums" >
+    <ItemGroup>
+      <ChecksumItems Include="$(PackagesOutDir)**/*$(ChecksumExtension)" />
+      <Project Include="publish.proj">
+        <PublishTarget>UploadChecksumsToAzure</PublishTarget>
+        <AdditionalProperties>%(Project.AdditionalProperties);ItemsToPublish=@(ChecksumItems);RelativeBlobPath=$(ChecksumsRelativePath)</AdditionalProperties>
       </Project>
     </ItemGroup>
   </Target>
+  
+  <Target Name="CreateChecksums">
+    <ItemGroup>
+      <ItemsToChecksum Include="@(CompressedFile)" />
+      <ItemsToChecksum Include="@(RuntimePackageFile)" />
+      <ItemsToChecksum Include="@(IdentityPackageFile)" Condition="'$(PublishRidAgnosticPackages)' == 'true'" />
```

Comment:
I don't think we need the nupkg's checksum'd.  No one downloads them from our storage account.  The Sdk doesn't have checksums for their nukgs either.

We just need archives and installers.